packages/core/contracts/oracle/implementation/SpamGuardIdentifierLib.sol

```diff
@@ -3,6 +3,7 @@ import "./AdminIdentifierLib.sol";
 library SpamGuardIdentifierLib {
     // Returns a UTF-8 identifier representing a particular spam deletion proposal.
     // The identifier is of the form "SpamDeletionProposal n", where n is the proposal id provided.
+    // Note that for ids> 10^10 the generated identifier will no longer be unique.
```

Comment:
The issue suggests that we limit the range of inputs or add a require.

Just OOC, how hard would it be to make the input type a uint32, which would guarantee it would always be < 10^11?

Alternatively, we could add a require.

Thoughts?

Reply:
Thanks for your suggestion, it totally makes sense to me. Changes added.